config/cesm/machines/config_pio.xml

```diff
@@ -109,13 +109,6 @@
     </values>
   </entry>
   -->
-<!--  <entry id="LND_PIO_REARRANGER">
```

Comment:
I have no idea what this commented-out block of xml code was for, but I thought I'd go ahead and just remove it rather than trying to figure it out. @jedwards4b (or @ekluzek ) feel free to chime in if you feel this is important to maintain. (For what it's worth, I see this block uncommented-out on the e3sm side.)

Reply:
I'm pretty sure this was something that was required at one point with PIO2. Since, we no longer use PIO2 with CLM it doesn't come into play. But, may if we even do start using it, unless PIO2 now has a default.